tests/testthat/test-target_market_share.R

```diff
@@ -967,3 +967,23 @@ test_that("Initial value of technology_share consistent between `projected` and
     class = "multiple_currencies"
   )
 })
+
+test_that("technology_share doesn't increase monotonically with time (#315)", {
```

Comment:
This description of the problem is likely imprecise because the test continues to fail after adding the patch by @Antoine-Lalechere. @Antoine-Lalechere how would you phrase it differently so I can adapt the test accordingly?